src/sass/components/_button.scss

```diff
@@ -19,6 +21,11 @@
   line-height: 24px;
   padding: 5px 10px;
   font-weight: 400;
+  display: none;
```

Comment:
W ten sposób wpływasz na wszystkie potencjalne przyciski z klasą `.btn-main-small` w projekcie, czyli od razu po dodaniu klasy są one chowane, nie jest to oczekiwany wynik